java/src/jmri/server/json/memory/JsonMemoryHttpService.java

```diff
@@ -48,7 +48,7 @@ public ObjectNode doGet(Memory memory, String name, String type, JsonRequest req
                 if (val instanceof jmri.IdTag) {
                     ObjectNode idTagValue = idTagService.doGet((jmri.IdTag) val, name, IDTAG, request);
                     data.set(VALUE, idTagValue);
-                } else if (val instanceof Reportable) {
```

Comment:
Please leave instanceof alone and change the cast instead.

Reply:
@rhwood has this right.  Reportable and reporter are not the same thing.  Reportable objects are returned by reporters.

Reply:
The method it's being passed to is expecting a Reporter, not a Reportable?

![8358](https://user-images.githubusercontent.com/39652002/79403563-308bc400-7f87-11ea-811f-7f4072c19ed5.png)

Reply:
> The method it's being passed to is expecting a Reporter, not a Reportable?
> 
> ![8358](https://user-images.githubusercontent.com/39652002/79403563-308bc400-7f87-11ea-811f-7f4072c19ed5.png)

It is entirely possible that there is an error here, but based on context, I would expect special handling for a Reportable object here, but I’ll defer to @rhwood who knows this code  better than I do.

Reply:
I missed that, there is no manager for `Reportable`s, but there are managers for `Reporter`s, so I withdraw my request for changes.